README.md

```diff
@@ -29,6 +29,12 @@ common Linux PKCS#11 implementations:
 
 This is NOT an official Google product.
 
+Additional make options
+ - PKCS11_LONGTYPE=32 - set CK_LONG/CK_ULONG size to int32_t/uint32_t. Normally set to long int, which is machine/compiler dependent.
+ - STRICT_P11=1 - set structures to packed, which tests against fully compliant PKCS11 implementations.
+   example:
+            make PKCS11_LONGTYPE=32 STRICT_P11=1
```

Comment:
nit: enclose in treble-backquotes

Reply:
I'm not clear on the read me...you want triple quotes around the section I added, no problem, but then you want single quotes around the identifiers? I'm thinking it would be confusing.
```
 - `PKCS11_LONGTYPE`=32 - set `CK_LONG`/`CK_ULONG` size to int32_t/uint32_t. Normally set to long int, which is machine/compiler dependent.
 - `STRICT_P11`=1 - set structures to packed, which tests against fully compliant PKCS11 implementations.
   example:
            make `PKCS11_LONGTYPE`=32 `STRICT_P11`=1
```